### ISSUE TITLE
Add an algorithm

### ISSUE THREAD

**Issue by Tanmay-901** [Dude-901/Cpp-Algorithms#1]
# Instructions
The program should be your own, no plagiarism shall be entertained. Try to add different PRs for different concepts so that reviewing them is easier and quicker. Make sure to name the program according to the concept/algorithm it tends to apply followed by your name.
Try to write the Time complexity (and space complexity too if possible) in the description.

- [ ] Checked that your commit is not already present.
- [ ] Checked your commit is following the contribution guidelines.

**Comment by Namanmittal0007** [comment]
@Dude-901  Can i add algorithms as .cpp files?

**Comment by Tanmay-901** [comment]
@Namanmittal0007 sure, you can. just please make sure to add comments wherever seems necessary, so that others may understand to code easily.

**Comment by Pratyushraj2121** [comment]
Can I do this using C Language?

**Comment by Tanmay-901** [comment]
@Pratyushraj2121 Yes you can, I've added the label. Though I'd advise you to make a separate folder for clarity.

**Comment by Pratyushraj2121** [comment]
Ya sure!

**Comment by Tanmay-901** [comment]
@Pratyushraj2121 I've merged the PR but I recommended to add a separate folder if you wanted to add in c, but since your code is in cpp, I request you to set it out in the home directory because I intend to make directories on the basis of operations and that would make it difficult to manage cpp programs both in the home directory and in the cpp separate folder. You shall add a directory for C programs if required.
PS: I'm not doing it myself so that your forked repo is also in the same commit version.

**Comment by Pratyushraj2121** [comment]
Okay. I made a different folder for C. Is it fine?

**Comment by Tanmay-901** [comment]
Yes a separate directory for C is welcomed, but the c++ programs are required in the home directory for now.

**Comment by kaustavy2k** [comment]
i would like to contribute to this by adding an a sorting algorithm in C

**Comment by Sindhu-321** [comment]
If the issue is open can I give it a try? I would like to add c++  codes
